packages/@react-native-windows/telemetry/src/telemetry.ts

```diff
@@ -442,6 +442,21 @@ export class Telemetry {
       data: codedError?.data ?? {},
     };
 
+    // Scrub any potential PII present in codedError.data array, as long as the data is a string.
+    if (Object.keys(codedErrorStruct.data).length > 0) {
```

Comment:
What types are possible to put in `data[field]`? Can we change the type of Data to be `Record<string, string>`? I'm worried this fix will leave us open to something else in the future, like a nested object (which the current `Record<String,any>` type currently allows for).

Reply:
We can then call sanitize on every field. And we can toString or toJSON any non-string data value.

Reply:
The alternative would be to create a `sanitizeAny()` which can recursively through the entire potential tree and call sanitize on every string it finds. then make sure we're doing that any place that we have an `any`

Reply:
I improved the fix by using recursion, so it now traverses all fields, inspecting arrays and nested objects appropriately, and sanitizes if the "leaf" value is a string.